package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,89 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=False, superposition=False):
     """Returns RMSD between two coordinate sets *a* and *b*.
 
     *a* and *b* are arrays of the coordinates of N atoms of shape N*3
     as generated by, e.g.,
     :meth:`MDAnalysis.core.AtomGroup.AtomGroup.coordinates`.
 
-    An implicit optimal superposition is performed, which minimizes
-    the RMSD between *a* and *b* although both *a* and *b* must be
-    centered on the origin before performing the RMSD calculation so
-    that translations are removed.
-
-    One can use the *center* = ``True`` keyword, which subtracts the
-    center of geometry (for *weights* = ``None``) before the
-    superposition. With *weights*, a weighted average is computed as
-    the center.
-
-    The *weights* can be an array of length N, containing e.g. masses
-    for a weighted RMSD calculation.
-
-    The function uses Douglas Theobald's fast QCP algorithm
-    [Theobald2005]_ to calculate the RMSD.
-
-    Example::
-     >>> u = Universe(PSF,DCD)
-     >>> bb = u.select_atoms('backbone')
-     >>> A = bb.coordinates()  # coordinates of first frame
-     >>> u.trajectory[-1]      # forward to last frame
-     >>> B = bb.coordinates()  # coordinates of last frame
-     >>> rmsd(A,B)
-     6.8342494129169804
-
-     .. versionchanged: 0.8.1
-        *center* keyword added
+    Parameters
+    ----------
+    a, b : numpy.ndarray
+        coordinates to align
+    weights : numpy.ndarray (optional)
+        use to compute weighted average
+    center : bool (optional)
+        subtract center of geometry before calculation. With weights given
+        compute weighted average as center.
+    superposition : bool (optional)
+        perform a superposition before calculating the RMSD.
```

Comment:
I think is was a mistake that the docs for the old version called it just superposition although it was only rotational superposition.